components/home-page/bal-action/action-bbe/kafka-consumer-producer.md

```diff
@@ -32,7 +32,7 @@ service on new kafka:Listener(kafkaEndpoint, consumerConfigs) {
         self.orderProducer = check new (kafkaEndpoint);
     }
 
-    remote function onConsumerRecord(Order[] orders) returns error? {
+    remote method onConsumerRecord(Order[] orders) returns error? {
```

Comment:
Should not be changed in the code snippets

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.